fe/fe-core/src/main/java/org/apache/doris/nereids/stats/FilterEstimation.java

```diff
@@ -76,7 +75,7 @@ private StatsDeriveResult calculate(Expression expression) {
 
     @Override
     public StatsDeriveResult visit(Expression expr, EstimationContext context) {
-        return new StatsDeriveResult(inputStats).updateBySelectivity(DEFAULT_INEQUALITY_COMPARISON_SELECTIVITY);
+        return inputStats.updateBySelectivity(DEFAULT_INEQUALITY_COMPARISON_SELECTIVITY);
```

Comment:
if this function generate new StatsDeriveResult, renam this function to withSelectivity

Reply:
done